### PR TITLE
[MIG] base_exception: Migration to 16.0

### PR DESCRIPTION
**Pull request by matiasperalta1**

### CODE REVIEW

File: base_exception/views/base_exception_view.xml [hunk 1]
```diff
@@ -39,6 +39,7 @@
                             <field name="description" />
                         </group>
                     </group>
+                    <field name="exception_type" invisible="1" />
```
Comment (sebalix) [inline base_exception/views/base_exception_view.xml:42]: Why it is needed? This field is already displayed in the group below.

Comment (matiasperalta1) [reply base_exception/views/base_exception_view.xml:42]: Before, when Odoo rendered a field, used to declare the invisible=1. As of this version, it no longer takes this into account for those users who are not part of this group. Later we see that the field is used in a domain and odoo understands that that field was never brought into view. The field is displayed into the group but for users who are not part of that group the field does not exist and that is where the error occurs. In order to this not to happen, the field without group must be called with invisible=1

**Review by aliciagaarzo — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Comment by sebalix** [comment]
@matiasperalta1 can you add `odoo_test_helper` dependency in the `test-requirements.txt` file to get the CI green?
https://github.com/OCA/server-tools/blob/14.0/test-requirements.txt

**Comment by matiasperalta1** [comment]
@sebalix Done

**Review by sebastienbeau — APPROVED** [review]
LGTM code review

/ocabot merge

**Pull request merged**